src/cartesian/ErrorBar.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @fileOverview Render a group of error bar
+*/
+import React, { Component, PropTypes } from 'react';
+import Layer from '../container/Layer';
+
+class ErrorBars extends Component {
```

Comment:
And the value of errorBar will change the domain of `YAxis` in a horizontal LineChart. We should calculate the value of errorBar before calculating the domain of `xAxisMap` and `yAxisMap`.